### DISCUSSION TITLE
Добавить пометку транка, с которого приходит звонок

### DISCUSSION THREAD

**Discussion by cblvnche** [mikopbx/Core#284] [Ideas (Идеи)]
Добавьте пожалуйста пометку транка, с которого приходит входящий звонок и с которого совершается исходящий звонок, чтобы это отображалось в CDR.

**Comment by boffart** [comment]
В CDR эта информация присутствует, в web интерфейс она не вынесена. В Web видно только номер, на который позвонил клиент "**DID**". 
Пока нет планов по развитию страницы истории звонков в web интерфефейсе АТС. 
В будущем планировали разработать дополнительный модуль с отчетами по звонкам.
